src/composer.json

```diff
@@ -22,8 +22,8 @@
         "slim/slim": "3.12.0",
         "symfony/config" : "v3.3.18",
         "symfony/http-foundation" : "v3.3.18",
```

Comment:
Please update the version for other symfony components as well.

Reply:
good point ... 👍